plugins/module_utils/util.py

```diff
@@ -366,7 +366,7 @@ def normalize_healthcheck(healthcheck, normalize_test=False):
                 continue
             if key in duration_options:
                 value = convert_duration_to_nanosecond(value)
-            if not value:
+            if not value and not (healthcheck['test_cli_compatible'] and key == 'test'):
```

Comment:
```suggestion
            if not value and not (healthcheck.get('test_cli_compatible') and key == 'test'):
```
`test_cli_compatible` is generally not there; only if this is called from the docker_container module.

Reply:
Not really familiar with python language I didn't know that will raise an error.